Userland/Libraries/LibIPC/Encoder.h

```diff
@@ -22,6 +23,8 @@ bool encode(Encoder&, T const&)
     VERIFY_NOT_REACHED();
 }
 
+bool encode(Encoder&, AK::Empty const&);
```

Comment:
Same here, `Encoder& operator<<(Empty);` on the `Encoder` class (which I think shouldn't need to be a `const&` as well)

Reply:
Did you try it? I did this locally and it works fine:
```c++
class Decoder {
    ErrorOr<void> decode(Empty&)
    {
        return {};
    }
};
```
```c++
class Encoder {
    Encoder& operator<<(Empty)
    {
        return *this;
    }
};
```
```c++
Empty empty {};
encoder << empty;
```
```c++
Empty empty {};
TRY(decoder.decode(empty));
```